src/components/GeoMap.js

```diff
@@ -434,32 +434,43 @@ export default class GeoMap {
       let minValue = attrValue.min;
       let maxValue = attrValue.max;
 
-      if (attrValue.min === 'minValue') {
```

Comment:
@melalj  - I couldn't tell what these lines did and didn't see any changes when removing them.  If they're still needed - lemme know!